src/prefect/logging/handlers.py

```diff
@@ -230,6 +230,8 @@ def prepare(self, record: logging.LogRecord) -> Dict[str, Any]:
             except ValueError:
                 flow_run_id = None
 
+        formatted_message = self.format(record)
```

Comment:
since you're calling this earlier, it's best if you remove the other `format` call on line 242 and use this precomputed message instead